arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -545,6 +545,213 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{cacheName}/firewallRules/": {
```

Comment:
I think there is an extra slash in the end that shouldn't be there, correct ?
"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{cacheName}/firewallRules**/**":